backend/migrations/Version20250917193007.php

```
@@ -0,0 +1,31 @@
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20250917193007 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Change last login to nullable';
+    }
+
+    public function up(Schema $schema): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)